tests/system/presenter/test_search_users_by_name_email.py

```diff
@@ -0,0 +1,277 @@
+        self.set_models(
+            {
+                "user/2": {
+                    "username": "user2",
+                    "email": "user2@test.de",
+                    "first_name": "first2",
+                    "last_name": "last2",
+                },
+                "user/3": {
+                    "username": "user3",
+                    "email": "userX@test.de",
+                    "first_name": "first3",
+                    "last_name": "last3",
+                },
+                "user/4": {
+                    "username": "user4",
+                    "email": "userX@test.de",
+                    "first_name": "first4",
+                    "last_name": "last4",
+                },
+            }
+        )
+
+    def test_search_fields_variations(self) -> None:
+        status_code, data = self.request(
+            "search_users_by_name_email",
+            {
+                "permission_type": UserScope.Meeting.value,
+                "permission_id": 1,
+                "search": [
+                    {
+                        "username": "user2",
+                    },
+                    {
+                        "email": "userX@test.de",
+                    },
+                    {
+                        "username": "user2",
+                        "email": "userX@test.de",
+                    },
+                ],
+            },
+        )
+        self.assertEqual(status_code, 200)
+        self.assertCountEqual(
+            data["user2/"],
+            [
+                {
+                    "id": 2,
+                    "first_name": "first2",
+                    "last_name": "last2",
+                    "email": "user2@test.de",
+                }
+            ],
+        )
+        self.assertCountEqual(
+            data["/userX@test.de"],
+            [
+                {
+                    "id": 3,
+                    "first_name": "first3",
+                    "last_name": "last3",
+                    "email": "userX@test.de",
+                },
+                {
+                    "id": 4,
+                    "first_name": "first4",
+                    "last_name": "last4",
+                    "email": "userX@test.de",
+                },
+            ],
+        )
+        self.assertCountEqual(
+            data["user2/userX@test.de"],
+            [
+                {
+                    "id": 2,
+                    "first_name": "first2",
+                    "last_name": "last2",
+                    "email": "user2@test.de",
+                },
+                {
+                    "id": 3,
+                    "first_name": "first3",
+                    "last_name": "last3",
+                    "email": "userX@test.de",
+                },
+                {
+                    "id": 4,
+                    "first_name": "first4",
+                    "last_name": "last4",
+                    "email": "userX@test.de",
+                },
+            ],
```

Comment:
Mmh, intuitively I'd thought that `username` and `email` are combined with an `AND`, so this should return nothing. But if the use case requires it, that's fine with me.

But we wanted to do CSV import validation with this as well and wanted to validate ALL of `first_name`, `last_name` and  `email`, right? Then we should do thi uniformly.

Reply:
nothing to do:D